src/flyte/_internal/imagebuild/docker_builder.py

```diff
@@ -250,13 +266,63 @@ async def handle(layer: Commands, _: Path, dockerfile: str) -> str:
 
 class WorkDirHandler:
     @staticmethod
-    async def handle(layer: WorkDir, context_path: Path, dockerfile: str) -> str:
+    async def handle(layer: WorkDir, _: Path, dockerfile: str) -> str:
         # cd to the workdir
         dockerfile += f"\nWORKDIR {layer.workdir}\n"
 
         return dockerfile
 
 
+def get_secret_commands(layers: typing.Tuple[Layer, ...]) -> typing.List[str]:
+    commands = []
+
+    def _get_secret_command(secret: str | Secret) -> typing.List[str]:
+        if isinstance(secret, str):
+            if not os.path.exists(secret):
+                raise FileNotFoundError(f"Secret file '{secret}' not found")
+            return ["--secret", f"id={hash(secret)},src={secret}"]
+        secret_env_key = "_".join([k.upper() for k in filter(None, (secret.group, secret.key))])
+        secret_env = os.getenv(secret_env_key)
+        if secret_env:
+            return ["--secret", f"id={hash(secret)},env={secret_env}"]
+        secret_file_name = "_".join([k for k in filter(None, (secret.group, secret.key))])
+        secret_file_path = f"/etc/secrets/{secret_file_name}"
```

Comment:
Are we hardcoding etc secrets or is this the within container path?

Reply:
hardcoded it as flytekit. https://github.com/flyteorg/flytekit/blob/1a7133ac5970bd57a5bb8744a5a453d086c06564/flytekit/configuration/__init__.py#L517
I can make it configurable if needed

Reply:
what does that mean, is this inside the container?